src/biz/collab/folder_view.rs

```diff
@@ -0,0 +1,162 @@
+use std::collections::HashSet;
+
+use collab_folder::Folder;
+use shared_entity::dto::workspace_dto::FolderView;
+use uuid::Uuid;
+
+pub fn collab_folder_to_folder_view(folder: &Folder, depth: u32) -> FolderView {
```

Comment:
This looks like method that could be implemented directly on `FolderView`.

Reply:
apparently `FolderView` is in `shared-entity` crate to share structure between server and client. putting the implementation as where you mentioned will mean that the `shared-entity` crate will have to depend on the `collab-folder` for `Folder`, which we want to avoid having.